blueprints/networking/glb-hybrid-neg-internal/landing-hub.tf

```diff
@@ -0,0 +1,247 @@
+    : null
+  )
+  prefix         = var.prefix
+  project_create = var.projects_create != null
+
+  services = [
+    "compute.googleapis.com",
+    "networkmanagement.googleapis.com",
+    # Logging and Monitoring
+    "logging.googleapis.com",
+    "monitoring.googleapis.com"
+  ]
+}
+
+################################################################################
+#                                  Networking                                  #
+################################################################################
+
+module "vpc_landing_untrusted" {
+  source     = "../../../modules/net-vpc"
+  project_id = module.project_landing.project_id
+  name       = "landing-untrusted"
+
+  routes = {
+    spoke1-r1 = {
+      dest_range    = var.vpc_spoke_config.r1_cidr
+      next_hop_type = "ilb"
+      next_hop      = module.nva_untrusted_ilbs["r1"].forwarding_rule_self_link
+    }
+    spoke1-r2 = {
+      dest_range    = var.vpc_spoke_config.r2_cidr
+      next_hop_type = "ilb"
+      next_hop      = module.nva_untrusted_ilbs["r2"].forwarding_rule_self_link
+    }
+  }
+
+  subnets = [
+    {
+      ip_cidr_range = var.vpc_landing_untrusted_config.r1_cidr
+      name          = "untrusted-${var.region_configs.r1.region_name}"
+      region        = var.region_configs.r1.region_name
+    },
+    {
+      ip_cidr_range = var.vpc_landing_untrusted_config.r2_cidr
+      name          = "untrusted-${var.region_configs.r2.region_name}"
+      region        = var.region_configs.r2.region_name
+    }
+  ]
+}
+
+module "vpc_landing_trusted" {
+  source     = "../../../modules/net-vpc"
+  project_id = module.project_landing.project_id
+  name       = "landing-trusted"
+  subnets = [
+    {
+      ip_cidr_range = var.vpc_landing_trusted_config.r1_cidr
+      name          = "trusted-${var.region_configs.r1.region_name}"
+      region        = var.region_configs.r1.region_name
+    },
+    {
+      ip_cidr_range = var.vpc_landing_trusted_config.r2_cidr
+      name          = "trusted-${var.region_configs.r2.region_name}"
+      region        = var.region_configs.r2.region_name
+    }
+  ]
+}
+
+module "firewall_landing_untrusted" {
+  source     = "../../../modules/net-vpc-firewall"
+  project_id = module.project_landing.project_id
+  network    = module.vpc_landing_untrusted.name
+
+  ingress_rules = {
+    allow-ssh-from-hcs = {
+      description = "Allow health checks to NVAs coming on port 22."
+      targets     = ["ssh"]
+      source_ranges = [
+        "130.211.0.0/22",
+        "35.191.0.0/16"
+      ]
+      rules = [{ protocol = "tcp", ports = [22] }]
+    }
+  }
+}
+
+module "nats_landing" {
+  for_each       = var.region_configs
+  source         = "../../../modules/net-cloudnat"
+  project_id     = module.project_landing.project_id
+  region         = each.value.region_name
+  name           = "nat-${each.value.region_name}"
+  router_network = module.vpc_landing_untrusted.self_link
+}
+
+module "nva_instance_templates" {
```

Comment:
Can you move nva resources into an `nva.tf` file?

Reply:
done